CONTRIBUTING.rst

```diff
@@ -4,8 +4,8 @@ Contributing
 Hi, thanks for your interest in the project! We welcome pull requests
 from developers of all skill levels.
 
-Jens H. Nielsen (j.h.nielsen@nbi.ku.dk), William H.P Nielsen
-(william@nbi.ku.dk), Dominik Vogel (a-dovoge@microsoft.com), and
+Jens H. Nielsen (Jens.Nielsen@microsoft.com), William H.P Nielsen
+(wihpniel@microsoft.com@nbi.ku.dk), Dominik Vogel (a-dovoge@microsoft.com), and
 Sohail Chatoor (a-sochat@microsoft.com) are the current maintainers of
```

Comment:
Mikhail should be mentioned here. @sohailc, I guess we should take you off the list now? 😢